website/docs/writing-docs.mdx

```diff
@@ -0,0 +1,176 @@
+
+# Hello from Docusaurus
+
+Are you ready to create the documentation site for your open source project?
+```
+
+<!-- TODO: talk about where to put the docs, resolving docs outside of website directory, etc. -->
+
+## Embedding React Components
+
+Docusaurus has in-house support for [MDX](https://mdxjs.com). This allows you to write JSX within your Markdown files and render them as React components.
+
+Rename your file from `.md` to `.mdx`. Docusaurus will hot reload your dev server and you won't need to restart.
+
+And try this block here:
+
+```jsx
+export const Highlighter = ({children}) => (
+  <span
+    style={{
+      background: 'rgb(37, 194, 160)',
+      padding: '2px 4px',
+      borderRadius: '2px',
+      color: 'white',
+    }}>
+    {children}
+  </span>
+);
+
+### <Highlighter>Docusaurus</Highlighter> green is my favorite color.
+```
+
+Notice how it renders both the markup from your React component and the Markdown syntax:
+
+export const Highlighter = ({children}) => (
+  <span
+    style={{
+      background: 'rgb(37, 194, 160)',
+      padding: '2px 4px',
+      borderRadius: '2px',
+      color: 'white',
+    }}>
+    {children}
+  </span>
+);
+
+<!-- TODO: Need a more standout typesetting for block quote. Using ### now -->
+
+### <Highlighter>Docusaurus</Highlighter> green is my favorite color.
+
+<!-- TODO: Briefly explain MDX more -->
+
+## Sidebar
+
+To generate a sidebar to your Docusaurus site, you need `sidebars.js` under the root directory. Our template should have already generated it for you, so you may edit it for your own customization.
+
+<!-- TODO: change classic template to use `sidebars.js` from json -->
+
+```bash
+.
+├── docs
+│   └── greeting.md
+├── sidebars.js
+```
+
+The `sidebars` object requires a `docs` field, under which the keys are the names of each category of your docs, and the values are arrays of `id`'s you specify in your docs' frontmatter.
+
+To add a doc to the sidebar, add the `id` specified in the frontmatter of the doc into its category.
+
+```diff
+module.exports = {
+  docs: {
++   "Getting started": ["hello"],
+    "Docusaurus": ["doc1"],
+    "First Category": ["doc2"],
+    "Second Category": ["doc3"],
+  }
+};
+```
+
+<!--
+TODO: Talk more about using the official docs plugin and how to configure the sidebar. Mention about incorporating assets and a preview of the cool Markdown features available, but don't list all the Markdown features here.
+References:
+- https://docusaurus.io/docs/en/navigation
+-->
+
+## Syntax highlighting
+
+Code blocks are text blocks wrapped around by strings of 3 backticks.
+
+    ```jsx
+      console.log("Hello world");
+    ```
+
+<!-- TODO: We need to allow users to pick syntax highlighting themes (maybe other than swizzling) -->
+
+Docusaurus uses Prism for syntax highlighting.
+
+```jsx
+console.log('Hello world');
+```
+
+## Live Editor
+
+You can create live code editors by creating a code block with `live` attached to the language meta string.
```

Comment:
i want to remove live editor from theme classic. let's remove the docs for this.

Reply:
its ok if u dont do it in the pr too. i'll remove it when i move live codeblock to plugin

Reply:
Oh, sure. Although, I put a note there already. Feel free to revise when you move it to plugin. Although, I find it worth mentioning there even if it's a plugin.